CHANGELOG.md

```diff
@@ -81,6 +81,9 @@ at locations that better optimize for object storage.
 
 - Introduced bootstrap command options to specify custom schema files for database initialization.
 
+- Added refresh credentials endpoint configuration to LoadTableResponse for AWS, Azure, and GCP. Enabling
+automatic credential refresh per table on the client side.
```

Comment:
nit: I understand the intent but i think imho its better to be explicit here, as there.
```suggestion
automatic storage credential refresh per table on the client side.
```